velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -162,7 +162,7 @@ TEST_F(ParquetTpchTest, Q6) {
   assertQuery(6, 2, 10);
 }
 
-TEST_F(ParquetTpchTest, Q18) {
+TEST_F(ParquetTpchTest, DISABLED_Q18) {
```

Comment:
Perhaps, add a comment explaining why the test is disabled.

Reply:
done.